docs/source/index.rst

```diff
@@ -66,7 +66,8 @@ This user guide contains:
    sci_api_tutorial/structure_selection
    sci_api_tutorial/capping
    sci_api_tutorial/armer
-   sci_api_tutorial/PDBParser
+   sci_api_tutorial/PDBParsersci_api_tutorial/PDBParser
+   sci_api_tutorial/Structure
```

Comment:
Let's make a folder called "class_reference" and put the page under "Documents". (i.e, create a new folder and move this line to line 78)